.packit.yml

```diff
@@ -51,6 +52,20 @@ jobs:
       # containers is very flaky
       - fedora-rawhide-x86_64
 
+  - job: tests
```

Comment:
Can we use the YAML merge key syntax to minimize configuration duplication?

https://packit.dev/docs/configuration/jobs

Reply:
I prefer the regular syntax, IMO this merge key syntax is unreadable if you never worked with it before...

Reply:
I agree that the merge key syntax is a bit hard to read initially. However, its still worth using it to reduce duplication and it can increase readability considering large .yml files.

Reply:
Let's decide to rewrite with merge syntax as soon this configuration gets more complicated than 2 entries